### ISSUE TITLE
Please create a tool to mass-swap items out of the game 

### ISSUE THREAD

**Issue by dmhenderson76** [DarkCastleMUD/DarkCastle#162]
Occasionally, we re-balance equipment and need a way to remove existing (custom) copies of those items from the game and replace them with the new version.  

Requirements:
* All items of vnum <x> must be removed from play including items in player inventory, worn items, vaulted items and items in consignment.
* Must be replaced with revised item (not sure if it makes sense to create new vnums here or just revise the old one)
* Players must not be able to use the old pre-toned item(s)

**Comment by bdunavant** [comment]
IIRC the way to do this, is to delete the VNUM and not re-use it.   When the user logs in, if the item number no longer exists, it will remove it from their inventory.   Create the toned version with a different item number, and then update it.

The way it works currently was specifically so we allow people to keep the old items even if we change future versions.

**Comment by dmhenderson76** [comment]
Yes, that's the way we currently do it but Jared suggested creating a tool to do this instead of just retiring vnums.  Also typically, we don't delete the vnum but remove the "custom" flag and all hit location flags but "take".  Jared already wrote code that will automatically return anything worn in an invalid location to inventory (along with a warning) on login.

**Comment by TehDreadPirateRoberts** [comment]
Kinda a one-off thing that hopefully we won't need to ever do again. Will revisit if a massive equipment re-work is done again.
